Silicon/CommonSocPkg/Include/Library/GpioLib.h

```diff
@@ -23,6 +23,11 @@
 //
 #define GPIO_PADCFG_DW_REG_NUMBER  4
 
+//
+// Max supported platform ID for GPIO base table
+//
+#define GPIO_MAX_BASE_TABLE_PLATFORM_ID  32
```

Comment:
Should this be 32 or 31 and the check use "<=" instead of "<"

Reply:
Checks for platform ID elsewhere in the source code use ``< 32`` , so I think keeping this as 32 will be better